app/Http/routes.php

```diff
@@ -26,6 +26,10 @@
     Route::model('competitions', 'Gladiator\Models\Competition');
     Route::resource('competitions', 'CompetitionsController');
 
+    // Competitions
+    Route::model('contest', 'Gladiator\Models\Contest');
```

Comment:
Also all of our views directories are plural as well (`/views/competitions/`, `/views/waitingrooms/`) so maybe update contests to be the same.